kubernetes/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+# create tmpdir
+
+mkdir -p /tmp/vault-snapshots
```

Comment:
Change to mount path of `emptyDir`.

Reply:
https://github.com/adfinis/vault-raft-backup-agent/pull/13/commits/f5d9b72984467fbce2443e5f8d654ec1a3a0e309
please re-review